README.md

```diff
@@ -37,8 +72,15 @@ The project is current divided into three repositories handling complementary as
   A Python backend ingests new captured HTML, computes diffs (for now, by
   querying PageFreezer), performs prioritization/filtering, and populates
   databases for Rails app.
+* [**web-monitoring-differ**](https://github.com/edgi-govdata-archiving/web-monitoring-differ)
```

Comment:
This breaks the "three" listed above. I'm wondering if we want to list this as an additional repo? This isn't part of the architecture being built ATM, right?